`Role` not exported

The `Role` type is not exposed by the library, which prevents me from using that type from inside my code.

I'm trying to map your role to our internal role type, but I can't access your role type
<img width="790" alt="image" src="https://github.com/edlink/edlink-node-sdk/assets/941075/66568910-a404-4d4f-a534-1957ba7ce1fd">

If I import from `@edlink/typescript/dist/types`, then it fails with the error:
```
Error: Cannot find module '@edlink/typescript/dist/types'
```

Ideally, anything that should be accessible by anyone consuming the library should be exposed in the [`index.ts`](https://github.com/edlink/edlink-node-sdk/blob/main/src/index.ts) file

--- a/package.json
+++ b/package.json
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.5",
+  "version": "1.0.6",
   "repository": "https://github.com/edlink/edlink-node-sdk",
   "license": "Unlicense",
   "main": "dist/index.js",

--- a/src/index.ts
+++ b/src/index.ts
@@ -5,21 +5,7 @@ import { IntegrationTokenSet, PersonTokenSet, TokenSetType } from './types';
 import { User } from './user';
 import { Auth } from './user/auth';
 
-export {
-    District,
-    School,
-    Session,
-    Course,
-    Class,
-    Section,
-    Person,
-    Enrollment,
-    Agent,
-    TokenSet,
-    PersonTokenSet,
-    IntegrationTokenSet,
-    TokenSetType
-} from './types';
+export * from './types';
 
 
 export type EdlinkConfig = {

